src/Core/src/Handlers/Image/ImageHandler.Windows.cs

```diff
@@ -8,33 +9,113 @@ namespace Microsoft.Maui.Handlers
 {
 	public partial class ImageHandler : ViewHandler<IImage, Image>
 	{
+		/// <inheritdoc/>
 		protected override Image CreatePlatformView() => new Image();
 
+		/// <inheritdoc/>
 		protected override void DisconnectHandler(Image platformView)
 		{
 			base.DisconnectHandler(platformView);
 			SourceLoader.Reset();
 		}
 
+		/// <inheritdoc/>
 		public override bool NeedsContainer =>
 			VirtualView?.Background != null ||
-			base.NeedsContainer;
+			VirtualView?.Aspect == Aspect.AspectFill;
```

Comment:
This should still _also_ check `base.NeedsContainer`; that way, it'll still work correctly if we introduce another class between this and ViewHandler.